tests/drivers/dma/loop_transfer/boards/adp_xc7k_ae350.conf

```diff
@@ -0,0 +1 @@
+CONFIG_DCACHE=n
```

Comment:
this is the n-th time I comment on this and I'm getting a bit tired. From the cache API prospective this makes no sense.

Why do you need to set this to `n`? This symbols is usually selected at soc / platform level to signify that a d-cache is present without telling anything whether this is used or not. `CONFIG_CACHE_MANAGEMENT` should be selected or not depending on whether you want to use the cache driver ops or not.

At driver / test / app level only `CONFIG_CACHE_MANAGEMENT` should be used. When `CONFIG_CACHE_MANAGEMENT` is set to `n` all the cache ops are NOPs, see: https://github.com/zephyrproject-rtos/zephyr/blob/7c418a065cfe483c464d9a6e2615afdad68568d1/include/zephyr/cache.h#L175-L181

I'm NACK-ing this until this cache mess is solved.

Reply:
Oh! I got it.
Yes, the I/DCACHE config shold not be selected by app layer. If someone want to disable the cache, he/she need to use the cache api.
And in this testcae we should enable the CONFIG_NOCACHE_MEMORY, not to disable the dcache.
Besides, our cache driver is on development, it might be useful someday.

We are so sorry for didn't catch your point before, and sincere appreciate for your patience.